models/processing.py

```diff
@@ -1,15 +1,43 @@
 # On the spot filtering. Different from the one-time preprocessing
-def load_dataset():
+# On the spot filtering. Different from the one-time preprocessing
+from datetime import datetime
+
+
+def load_dataset(csv):
     """
     Load dataset from the preprocessed CSV
     :return:
     """
-    pass
+    df = pd.read_csv(csv)
+    return df
 
 
 def get_day_of_week(date):
-    pass
+    day_of_week = date.strftime('%A')
+    return day_of_week
+
```

Comment:
There is some inconsistent formatting throughout. You can press "ctrl+alt+l" in PyCharm to automatically format the file